packages/OSCADml/OSCADml.0.2.0/opam

```diff
@@ -17,7 +17,7 @@ depends: [
   "ocaml" {>= "4.14.0"}
   "gg" {>= "1.0.0"}
   "cairo2" {>= "0.6.2"}
-  "OCADml" {>= "0.3.0"}
+  "OCADml" {= "0.3.1" | = "0.3.2"}
```

Comment:
Why not
```suggestion
  "OCADml" {>= "0.3.0" & < "0.3.3"}
```
It was working on 0.3.0 right? And this allows for possible bugfix backports (no matter how improbable)

Reply:
Updated with this suggestion, except with a lower bound of `0.3.1`, as `0.3.0` has been marked unavailable since my attempted hotfixing debacle.